spec/legacy_uuid_spec.rb

```diff
@@ -58,6 +58,15 @@
     end
   end
 
+  context ".from_organiser uuid format" do
+    it "uses a predefined prefix" do
```

Comment:
You already have a context dedicated to `.from_organiser` above, so rename this test to something like `it "transparently handles UUIDs"` and merge it into the block above.